pkgs/by-name/gi/gitlab-release-cli/package.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, fetchFromGitLab
+, buildGoModule
+}:
+
+buildGoModule rec {
+  pname = "gitlab-release-cli";
+  version = "0.18.0";
+
+  src = fetchFromGitLab {
+    owner = "gitlab-org";
+    repo = "release-cli";
+    rev = "v${version}";
+    hash = "sha256-CCSice/uMf2OfFNEpwwhX6A0wrSsC1v9XWEhAAwQRso=";
+  };
+
+  vendorHash = "sha256-UwDMRsWbk8rEv2d5FssIzCLby68YZULoxd3/JGLsCQU=";
+
+  meta = {
+    description = "GitLab toolset to create, retrieve and update releases";
```

Quoting the section [Meta attributes in `pkgs/README.md`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes):
> `meta.description` must:
> …
> ○ Not start with the package name.
>   ■ More generally, it should not refer to the package name.


```suggestion
    description = "Toolset to create, retrieve and update releases";
```

---

This change doesn't look appropriate. We should state that it applies to GitLab.

---

How is this, @Aleksanaa? 🙂 

```suggestion
    description = "Toolset to create, retrieve and update releases on GitLab";
```

---

I'm not sure what the difference is between this change and the previous one. It even seems a bit more verbose

---

Thank you for the discussion,  I applied the last description.